pkg/services/sqlstore/migrations/ualert/alert_rule.go

```diff
@@ -215,7 +219,9 @@ func transExecErr(s string) (string, error) {
 	case "", "alerting":
 		return "Alerting", nil
 	case "keep_state":
-		return "Alerting", nil
```

Comment:
To match my comment above, this wouldn't be necessary and we wouldn't incur a breaking change during the migration.